### ISSUE TITLE
Strange behavior using IE browser

### ISSUE THREAD

**Issue by kavitama** [krispo/angular-nvd3#226]
While being able to show graphs without any specific issue on Opera, Firefox, Chrome, Safari when the same code is executed on Internet Explorer Browser the graph has missing parts (CSS problem?) like in the example attached.
Any hint on where the issue could come from?
![screenshot048](https://cloud.githubusercontent.com/assets/5692413/9885836/9d8eb44c-5bbd-11e5-8314-6d71bac1634c.jpg)
![screenshot047](https://cloud.githubusercontent.com/assets/5692413/9885840/a3c5e9ac-5bbd-11e5-84f3-68c0cd6a5dea.jpg)
![screenshot049](https://cloud.githubusercontent.com/assets/5692413/9885843/a7cb3494-5bbd-11e5-8104-c702cbfafc8c.jpg)

**Comment by EvilDrW** [comment]
I looked into this and it seems to be a CSS problem, yes.  When I manually edit the <svg> container of the charts and change it from (for example):
```html
<svg class="nvd3-svg" width="100%" height="450">
```
to:
```html
<svg class="nvd3-svg" style="width:100%; height:450px">
```

then it renders correctly.  I guess I would recommend using the style attribute rather than width/height attributes?  note that this is the same problem as #200 experiences

**Comment by kavitama** [comment]
![image](https://cloud.githubusercontent.com/assets/5692413/10046192/a712af20-61dd-11e5-8c6c-86b9dfc70aab.png)
Not sure if it is the same but your proposal would be to change where the height is given ?
In the <nvd3-multi-bar-chart> ???

**Comment by EvilDrW** [comment]
Yes, I was able to solve this by changing at angular-nvd3.js#L170 to
```js
d3.select(element[0]).append('svg')
   .style({height: scope.options.chart.height+'px', width: scope.options.chart.width || '100%'})
   .datum(data)
   .transition().duration(scope.options.chart.transitionDuration)
   .call(scope.chart);
```
I'll submit a PR this evening when I'm not on a hobbled work pc.

**Comment by krispo** [comment]
Thanks guys for your investigation into this. I also added some [fixes](https://github.com/krispo/angular-nvd3/commit/42d807c663d77c594026d6426ac078e0d30dec8e) into your PR. The issue will be fixed in the new release (1.0.3)
